src/js/handlebars.helpers.js

```diff
@@ -124,15 +124,35 @@ export default function handlebarsHelpers(handlebars) {
    *
    * Usage:
    * {{formatDuration duration}}
-   * {{formatDuration duration long}}
+   * {{formatDuration duration "long"}}
    */
   handlebars.registerHelper('formatDuration', function(duration, format) {
+    // Return empty string when there is no duration.
     if (!duration) {
       return "";
     }
-    const {hours = "", minutes = "", seconds = ""} = duration;
+
+    // Nothing to process here when the duration is already in short format string
+    // (to support existing CMS metadata).
+    if (typeof(duration) === 'string' && format !== "long") {
+      return duration;
+    }
+
     let durationString = "";
     let parts = [];
+    // Support for object type 'duration'.
+    let {hours = "", minutes = "", seconds = ""} = duration;
```

Comment:
is this safe to do if duration is not an object?

Reply:
and should this be part of the else of if (typeof(duration) === 'string') {

Reply:
Yes, the destructuring assignment works for string, and it will assign default value of empty string. 
Point taken, though. I've updated the code to be more readable.